src/cdi/CDI.hh

```diff
@@ -511,6 +511,9 @@ class CDI {
   //! Material ID.
   std_string matID;
 
+  //! Extend integration to place low and high tails in low and high groups?
+  static bool extend;
```

Comment:
Sigh. Global variables always seem to cause problems.  Can you try prepending this line with `DLL_PUBLIC_cdi`? See `c4/C4_MPI.hh` for an example (search for `DLL_PUBLIC_c4`).

Reply:
Done.

Reply:
If this PR hits additional MSVC issues, I'll checkout your branch and work through the necessary changes.  With any luck it won't come to this.